app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+class CommandError(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3:
+        raise CommandError(f"Wrong format of command: {command}")
+
+    if command[0] != "mv":
+        raise CommandError(f"Wrong command:"
+                           f"'{command[0]}', did you mean 'mv'?")
+
+    file_name, full_path = command[1], command[2]
```

Comment:
In my opinion " _, file_name, full_path = command" looks more readable.
Also, try not to use indexing (command[0]) if it is possible

Reply:
I don't think I need to write unnecessary `_` here, it has no impact on logic makes additional link to object that I will never use and less readable
And for no using command[0] I just don't want to create new variable to use it once